### PR TITLE
Add For Enterprises section, new design, and some other changes

### PR DESCRIPTION
**Pull request by Vinnl**
The noteworthy changes are:

- Addition of [For enterprises](https://solid.github.io/for-enterprises) pages
- Target audience-specific overview of what makes Solid attractive ("Why you'll love Solid")
- More extensive documentation on common vocabularies and how to publish your own
- Updated FAQs
- _Many_ spelling corrections and link fixes (thanks @RubenVerborgh!)
- Updated design, courtesy of Lisa

### CODE REVIEW

**Comment by Vinnl** [comment]
Inrupt is currently revisiting its approach to website content, so I'm putting this on hold for a bit while a conclusion is reached there, so that the design rework from #103 can be deployed to the draft website at solid.github.io in that time.

**Comment by Vinnl** [comment]
There's likely going to be a review session that might include this PR, so I'm re-opening it in case.

**Comment by RubenVerborgh** [comment]
Additionally assigning @Mitzi-Laszlo and @NSeydoux: can we please get this approved by @timbl? This PR fixes important copy on the website.

File: _posts/this-week-in-solid/next.md [hunk 1]
```diff
@@ -1,34 +1,88 @@
...
 ---
 
 {% include twis-intro.md %}
 
-## [Press](https://solidproject.org/press)
+## Updates
+
+### [Press](https://solidproject.org/press)
+
+[Jon Henshaw](https://twitter.com/henshaw) published an article about the work Angelo Veltrans has been doing. [Read more here](https://www.coywolf.news/social/solid-profile-design/).
```
Comment (RubenVerborgh) [inline _posts/this-week-in-solid/next.md]: @Mitzi-Laszlo Angelo Veltens

File: _posts/this-week-in-solid/next.md [hunk 2]
```diff
@@ -1,34 +1,88 @@
...
+Have you seen any articles or talks about Solid this week? If you've seen any talks, articles, or written blog posts about Solid please do send them over for next week. 
+
+### Solidproject.org
 
-2020-02-07 [This CERN internal event](https://indico.cern.ch/e/CERN-Solid-brainstorming) may bring the place where Sir Tim invented the Web to the heart of the Web's future by Maria Dimou
+The landing page has been redesigned, and most other pages have received a small touch-up. Additionally, most content has undergone editing to ensure the information is relevant and free of typos. All in all, the starting point for all things Solid should be more pleasant to use now.
 
-Have you seen any articles or talks about Solid this week? If you've seen any talks, articles, or written blog posts about Solid please do send them over for next week.
+### [Developer Tools](https://solidproject.org/for-developers/apps/tools)
+* [Mark Hughes](https://github.com/theWebalyst) is working on [showing how to use various RDF libs with Sveltejs](https://github.com/theWebalyst/svelte-with-rdf-libs-webpack) and has rdflib.js and graphy working
```
Comment (RubenVerborgh) [inline _posts/this-week-in-solid/next.md]: @Mitzi-Laszlo It's Svelte.

File: _posts/this-week-in-solid/next.md (hunk 1 above)
Comment (Vinnl) [reply _posts/this-week-in-solid/next.md]:
```suggestion
[Jon Henshaw](https://twitter.com/henshaw) published an article about the work Angelo Veltens has been doing. [Read more here](https://www.coywolf.news/social/solid-profile-design/). 
```

File: _posts/this-week-in-solid/next.md (hunk 2 above)
Comment (Vinnl) [reply _posts/this-week-in-solid/next.md]:
```suggestion
* [Mark Hughes](https://github.com/theWebalyst) is working on [showing how to use various RDF libs with Svelte](https://github.com/theWebalyst/svelte-with-rdf-libs-webpack) and has rdflib.js and graphy working
```

**Comment by Mitzi-Laszlo** [comment]
Looks like when I added to the next.md of TWiS that the review is automatically impacted?

**Comment by RubenVerborgh** [comment]
We can just merge up to the point Tim approved in order to still adhere to the process. Let me do that, then next changes become a new PR.

**Comment by RubenVerborgh** [comment]
Thanks all, set `master` to `b843c4acdbc9bf0cd9d8e79bb4394c8a82aa21ad`, which is the latest commit Tim approved.

Future changes can happen as a new pull request from `staging` to `master`.

**Comment by Vinnl** [comment]
Hmm, it's a bit weird that @RubenVerborgh can merge to `master` given that you're not in the [Creators](https://github.com/solid/process/blob/master/creators.md) - if something had gone wrong with the deploy, I don't think you'd have the required access to fix it?

Anyway, happy that this is finally live :)

**Comment by RubenVerborgh** [comment]
Given that I'm the one who set up the current deployment, things would be pretty bad if I couldn't fix it 😛 Yay for the live site!

**Pull request closed without merging**